src/clm/commands/collapse_carbon_files.py

```diff
@@ -0,0 +1,37 @@
+from clm.functions import read_csv_file, write_to_csv_file
+
+
+def add_args(parser):
+    parser.add_argument(
+        "--carbon_files",
+        type=str,
+        nargs="+",
+        help="Input file paths for compressed carbon files",
+    )
+    parser.add_argument(
+        "--output_file",
+        type=str,
+        help="File path to save the all the compressed carbon files",
+    )
+    return parser
+
+
+def collapse_carbon_files(carbon_files, output_file):
+    carbon_files = sorted(carbon_files)
```

Comment:
Since we're sorting the lines later anyway, I don't think this sort is necessary.